projects/components/src/data-exporter/data-exporter.component.html

```diff
@@ -65,7 +65,9 @@ <h3 class="modal-title">{{ dialogHeader | lazyString }}</h3>
                                         <clr-checkbox-wrapper class="column-checkbox">
                                             <input type="checkbox" clrCheckbox [formControlName]="col.fieldName" />
                                             <label>{{
-                                                this.friendlyFieldsControl.value ? col.displayName : col.fieldName
+                                                this.friendlyFieldsControl.value
```

Comment:
Shouldn't this just be `col.displayName || col.fieldName` ?

Reply:
No, because in the case that this.friendlyFieldsControl.value is true, we want to display the `col.displayName` with a backup of `col.fieldName`, but in the case it is false, we only want to display `col.fieldName`

Reply:
Should be calling getDisplay name instead of duplicating the defaulting